traffic_ops/app/db/migrations/20170315000001_user_tenancy.sql

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
can you change the name of this file? it is not longer the "latest" migration. 20170330000001_user_tenancy.sql should work.